core/cluster/process.go

```diff
@@ -0,0 +1,38 @@
+package cluster
+
+import "time"
+
+type (
+
+	SingleProcess interface {
```

Comment:
Rename to `Once`?

Reply:
Once and Locker works because of using same instance of the type. Which is replaced by the key paramter. so it cannot be interface compatible but the renaming makes sense